packages/components/src/angle-picker-control/index.tsx

```diff
@@ -83,32 +88,18 @@ function UnforwardedAnglePickerControl(
 					step="1"
 					value={ value }
 					spinControls="none"
-					suffix={
-						<Spacer
-							as={ Text }
-							marginBottom={ 0 }
-							marginRight={ space( 3 ) }
-							style={ {
-								color: COLORS.ui.theme,
-							} }
-						>
-							°
-						</Spacer>
-					}
+					prefix={ prefixedUnitText }
+					suffix={ suffixedUnitText }
 				/>
 			</FlexBlock>
-			<FlexItem
-				style={ {
-					marginBottom: space( 1 ),
-					marginTop: 'auto',
-				} }
-			>
+			<Spacer marginBottom="1" marginTop="auto">
 				<AngleCircle
+					refNumberInput={ refNumberInput }
```

Comment:
Nit: Would it be cleaner to pass in an `onDragStart` callback rather than the raw ref?

```suggestion
					onDragStart={ () => refNumberInput.current?.focus() }
```

Reply:
I think so. Thank you!